pkg/proc/proc_test.go

```diff
@@ -4979,3 +4982,166 @@ func TestStepOutPreservesGoroutine(t *testing.T) {
 		}
 	})
 }
+
+func TestDump(t *testing.T) {
+	if runtime.GOOS == "freebsd" || (runtime.GOOS == "darwin" && testBackend == "native") {
+		t.Skip("not supported")
+	}
+	if os.Getenv("CI") == "true" && buildMode == "pie" {
```

Comment:
Does this need to be updated for TeamCity CI?

Reply:
I've updated the skip message but it's still occasionally failing. I'm leaning towards this being caused by a legitimate bug, but I can't reproduce it locally. Maybe it has something to do with docker.